client/src/components/Page.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { t } from "@lingui/macro";
+import { withI18n, withI18nProps } from "@lingui/react";
+import helpers from "../util/helpers";
+
+const metadata = {
+  keywords: t`Landlord, Portfolio, Tenant, Displacement, Map, JustFix, NYC, New York, Housing, Who Owns What`,
+  description: t`Use this free tool from JustFix.nyc to research your building and investigate landlords! We use property ownership mapping to identify a landlord's portfolio and provide data to indicate potential tenant harassment and displacement.`,
+  siteName: t`Who owns what in nyc?`,
+};
+
+export const FB_APP_ID = "247990609143668";
+const TWITTER_HANDLE = "@JustFixNYC";
+const ORIGIN_URL = "https://whoownswhat.justfix.nyc/";
+const SHARE_IMAGE_URL = "https://i.imgur.com/6WL74DZ.png";
+
```

Comment:
Saving these as constants triggered me to record some issues to address later: #268, #269, and #270

Reply:
nooooooooooooooice i dig it!!  Also I find the term `ORIGIN_URL` a tiny bit confusing b/c I think of the word "origin" in the same sense as the [`Origin` HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin), which contains no path information, so I would expect something named `ORIGIN_URL` to not contain the trailing slash at the end.

Reply:
Ahh yeah, I don't think there's any reason why we need to have that trailing slash there. Let me remove it.